test/interop/C/makefiles/dynamicLinking/SKIPIF

```diff
@@ -0,0 +1,10 @@
+CHPL_COMM != none
+
+# Some (probably all, haven't doublechecked) linkers will refuse to create
+# shared libraries when there are _any_ text relocations present.
+# The reason being that if you have a text relocation, it means that your
+# shared library isn't fully position independent. If your SO isn't PIC, then
+# indepedent processes can't share a single copy of its text segment.
```

Comment:
The problem isn't that "independent processes can't share a single copy of its text segment", it's that if any non-PIC code is present in a shared object then that shared object has to be loaded at the exact same address in all processes which use it, and runtime loaders don't typically allow such a constraint.

Reply:
I thought that it was more along the lines of "if we have text relocations, then when it comes time to load we will have to load an independent copy of the text segment into the address space of each process and perform relocations on it". Whereas if it's PIC, then all processes can share the same text segment and only have to duplicate the data segment.

So what you're saying is that if there are text relocations in a shared library, the loader will be forced to have all processes load in the shared library at the same virtual address (instead of what I thought happens)?

Reply:
Aha, I was wrong.  The last time I was working closely with a dynamic linker, there were only certain relocation types it was willing to do, and those didn't include regular text relocations.  If it saw such a relocation type in a shared object it was loading, it just threw an error.

It seems that's not true any more.  The Linux dynamic linker can do text relocations even for a shared object built non-PIC, with the main bad result being that every process using the SO will have its own private copy.  (It appears there are some other caveats, but those don't affect all programs.)

Given this, I think the original comment can stay.